README.md

```diff
@@ -150,6 +150,19 @@ been written into the [DAG Service][]'s storage mechanism.
 The input's file paths and directory structure will be preserved in the DAG
 Nodes.
 
+### Importer options
+
+In the second argument of the importer constructor you can specify the following options:
+
+* `chunker` (string, defaults to `"fixed"`): the chunking strategy. Now only supports `"fixed"`
+* `chunkSize` (positive integer, defaults to `262144`): the maximum chunk size for the `fixed` chunker.
```

Comment:
I think it would be better to have a specific `chunkerOptions` object, which has in the case for the fixed chunker an option of `size`, as for other chunkers this might not make sense as an option.